request/process.go

```diff
@@ -23,14 +29,17 @@ func Process() {
 
 	switch request.Action {
 	case "configure":
-		break
+		configure(request)
 	case "list":
 		break
 	case "fetch":
 		break
 	default:
 		log.Errorf("Received a browser request with an unknown action: %+v", request)
-		response.SendError(errors.CodeInvalidRequestAction, "Invalid request action", &map[string]string{"action": request.Action})
+		response.SendError(
+			errors.CodeInvalidRequestAction,
+			"Invalid request action",
+			&map[string]string{"action": request.Action})
```

Comment:
Can multiline function calls like this have the final `)` on a new line? Makes things a bit more readable.

Reply:
Sure, and done 👍